### PR TITLE
Update ci

### PR DESCRIPTION
**Pull request by shokmaster**
Update packages and add CI tools.

DB-147634

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "@semantic-release/npm",
+        "@semantic-release/git"
+    ],
+    "verifyConditions": [
+        "@semantic-release/changelog",
+        "@semantic-release/npm",
+        "@semantic-release/git",
+        "@semantic-release/github"
+    ]
+}
```
Comment (adrigzr) [inline .releaserc:10]: Try to update to new config style:

```
{
    "plugins": [
        "@semantic-release/commit-analyzer",
        "@semantic-release/release-notes-generator",
        "@semantic-release/changelog",
        "@semantic-release/npm",
        "@semantic-release/git",
        "@semantic-release/github"
    ]
}
```

File: package.json [hunk 2]
```diff
@@ -1,53 +1,88 @@
...
-  "description": "The default blueprint for ember-cli addons.",
+  "version": "1.0.0",
+  "description": "An ember-cli addon to manage notifications.",
   "keywords": [
-    "ember-addon"
+    "ember-addon",
+    "notifications",
+    "alerter"
   ],
+  "repository": "",
```
Comment (adrigzr) [inline package.json]: Fill in `repository`.

**Review by adrigzr — CHANGES_REQUESTED** [review]

File: .releaserc (hunk 1 above)
Comment (shokmaster) [reply .releaserc:10]: done!

File: package.json (hunk 2 above)
Comment (shokmaster) [reply package.json]: done!

**Review by adrigzr — DISMISSED** [review]

**Review by josex2r — APPROVED** [review]

**Pull request merged**